java/ql/test/kotlin/library-tests/ministdlib/MiniStdLib.kt

```diff
@@ -0,0 +1,33 @@
+package kotlin
+
+/*
+This is a mini standard library replacement, to make it easy to write
+very small tests that create very small databases.
+
+If you define a class, then you will need to also define any members that
+compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/descriptors/IrBuiltInsOverDescriptors.kt
+expects (e.g. with findFunctions(...).first) to exist.
+*/
+
+public open class Any {
+    fun toString(): String { return this.toString() }
+    open operator fun equals(other: Any?): Boolean { return this.equals(other) }
+}
+
+public open class String {
```

Comment:
Why are `String`, `Boolean`, and `Int` declared as `open`?

Reply:
Ah, I'd just made everything an open class without checking. Fixed, thanks!